src/mlpack/core/data/string_cleaning.hpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file string_cleaning.hpp
+ * @author jeffin sam
+ *
```

Comment:
We should add the description.
```
Definition of the StringCleaning class.
```

Reply:
@jeffin143 Why did you mark this issue as resolved? I don't see the fix.

Reply:
https://github.com/mlpack/mlpack/commit/03943e97778da97e822a17bd6a1885b90682c1b0

I did add, sorry is it not correct